packages/studio-be/src/core/bots/bot-service.ts

```diff
@@ -63,7 +63,13 @@ const BotCreationSchema = Joi.object().keys({
       id: Joi.string()
     }
   },
-  locked: Joi.bool()
+  locked: Joi.bool(),
+  isCloudBot: Joi.bool().optional(),
+  cloud: Joi.object({
+    oauthUrl: Joi.string().optional(),
+    clientId: Joi.string().optional(),
```

Comment:
A client Id has a fixed size (UUID), so maybe we should improve the validation we make here. What do you think?